gruntfile.js

```diff
@@ -134,7 +158,8 @@ module.exports = function (grunt) {
       'npm-contributors',
       'bump:' + (type || 'patch') + ':bump-only',
       'build',
-      'changelog',
+      'conventionalChangelog',
+      'conventionalGithubReleaser',
```

Comment:
This should probably run after `bump-commit`, as at this point the changelog isn't commited yet and no tag created